pgcli/main.py

```diff
@@ -472,7 +473,7 @@ def handle_editor_command(self, cli, document):
             cli.application.pre_run_callables = saved_callables
         return document
 
-    def execute_command(self, text, query):
```

Comment:
I believe we still need to pas in the `query` arg. 

Line 535 returns the `query` var, but it can go uninitialized if a user hits Ctrl-C during the execution of a command.

I can show you what you mean when we meet in the sprint room.